.gitignore

```diff
@@ -26,6 +26,9 @@ website/node_modules
 
 website/vendor
 
+# exclude localbuild
+terraform-provider-ovh
```

Comment:
if you use "make build" command, it will generate the go binary in `$GOPATH/bin/`, not in this folder ;-)

Reply:
agreed but there is no guarantee that user will use make build.
I don't see any downside to adding this to .gitignore